README.rst

```diff
@@ -5,3 +5,49 @@ Socraticqs2 is the web engine for courselets.org.
 
 Developer documentation is available at http://cjlee112.github.io/socraticqs2
 
+
+In settings/local_conf.py we need to describe all needed KEY/SECRET for social-auth:
```

Comment:
Should probably change to "For a developer / test version to access social-auth, you will need to add the following social-auth keys to settings/local_conf.py in your development install:"